src/styles/masked.css

```diff
@@ -0,0 +1,73 @@
+.masked-overflow {
```

Comment:
No choice here, forced to use a CSS file. Some of the CSS commands don't exist in tailwindcss at the moment.